fullstack-cert/html-projects/lab-movie-review-page/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movie Review</title>
+</head>
+<body>
+        <section class="review">
+            <h1>Movie Review: Rise Beyond</h1>
+            <img src="rise-beyond-2.webp" alt="Rise Beyond Movie Cover">
+            <p>
+                Rise Beyond is an uplifting drama that tells the story of Ethan Carter, 
+                a young man from a small town who dreams of becoming a world-class musician. 
+                Facing financial struggles and self-doubt, Ethan embarks on a journey of resilience, 
+                guided by his mentor Professor Adams and childhood friend Lena Mitchell. 
+                Through setbacks, failures, and moments of triumph, Rise Beyond reminds us that 
+                our greatest potential lies beyond our fears.
+            </p>
+
+            <label for="rating">Movie Rating:</label>
+            <meter id="rating" min="0" max="10" value="9.2" aria-labelledby="rating"></meter>
```

Comment:
I don't think the `meter` element was covered.
You will need to double check that in the prior modules
https://www.freecodecamp.org/learn/full-stack-developer/

if not, then you will need to remove it